ili2mssql/test/java/ch/ehi/ili2mssql/EnumTest23.java

```diff
@@ -0,0 +1,18 @@
+package ch.ehi.ili2mssql;
+
+import ch.ehi.ili2db.AbstractTestSetup;
+import ch.ehi.ili2db.Enum23Test;
+
+public class EnumTest23 extends Enum23Test {
```

Comment:
Please rename to a name with suffix Test

Reply:
Done!